privateGPT.py

```diff
@@ -48,17 +52,17 @@ def main():
         res = qa(query)
         answer, docs = res['result'], [] if args.hide_source else res['source_documents']
 
+        # Print the relevant sources used for the answer
+        for document in docs:
+            print("\n> " + document.metadata["source"] + ":")
+            print(document.page_content)
```

Comment:
Maybe adding some nice colors? Also using `deep-translator==1.11.1` with the env flag if someone want's to translate the answer?

translate_src = os.environ.get('TRANSLATE_SRC_LANG', "en")
translate_dst = os.environ.get('TRANSLATE_DST_LANG', "fr")

```python
for document in docs:
    print(f"\n\033[31m Source: {document.metadata['source']} \033[0m")
    if translate_ans:
        document.page_content = GoogleTranslator(source=translate_src, target=translate_dst).translate(document.page_content)
        print(f"\033[32m\033[2m : {document.page_content} \033[0m")
```

Reply:
Yes, I was already thinking about it. I was going to implement it as soon as I had some free time :)